DTW/Bridge DTF/map.json

```diff
@@ -0,0 +1,96 @@
+{
+	"name": "Bridge DTF",
+	"authors": ["Clijmart"],
+	"version": "1.0",
+	"gametype": "DTM",
+	"teams": [
+		{
+			"id": "yellow",
+			"name": "Benny",
+			"color": "yellow",
+			"max": 50,
+			"min": 1
+		},
+		{
+			"id": "gold",
+			"name": "Lucas",
+			"color": "gold",
+			"max": 50,
+			"min": 1
+		}
+	],
+	"spawns": [
+		{ "teams": ["spectators"], "coords": "0.5, 51, 0.5, 0" },
+		{ "teams": ["yellow"], "coords": "55.5, 27, -36.5, 45" },
+		{ "teams": ["gold"], "coords": "-54.5, 27, 37.5, -135" }
+	],
+	"dtm": {
+		"monuments": [
+			{
+				"name": "Lucas",
+				"teams": ["gold"],
+				"materials": ["wool"],
+				"region": {
+					"min": "-67, 62, 8",
+					"max": "-73, 56, 2"
+				},
+				"health": 20
+			},
+			{
+				"name": "Benny",
+				"teams": ["yellow"],
+				"materials": ["wool"],
+				"region": {
+					"min": "67, 62, -8",
+					"max": "73, 56, -2"
+				},
+				"health": 20
+			}
+		]
+	},
+	"killstreaks": [{
+			"count": 1,
+			"repeat": true,
+			"commands": ["give %killername% minecraft:golden_apple 1",
```

Comment:
Put the "[" on its own separate line since you added more than 1 command to the killstreaks.